queries/locals.scm

```diff
@@ -10,4 +12,16 @@
 (record_pattern_argument pattern: (identifier) @local.definition)
 
 ; References
-(identifier) @local.reference
+(field_access
+  record: (identifier) @local.reference
+  field: (identifier) @local.reference)
```

Comment:
Here I don't think `field` can be a local reference as it's the `bar` part of `foo.bar`.